packages/perseus/src/widgets/interactive-graphs/axis-labels.tsx

```diff
@@ -0,0 +1,58 @@
+import {vec} from "mafs";
+import React from "react";
+
+import {getDependencies} from "../../dependencies";
+
+import {pointToPixel} from "./graphs/use-transform";
+import useGraphConfig from "./reducer/use-graph-config";
+
+export default function AxisLabels(props) {
+    const {range, labels, width, height} = useGraphConfig();
+
+    const yAxisLabelLocation: vec.Vector2 = [0, range[1][1]];
+    const xAxisLabelLocation: vec.Vector2 = [range[0][1], 0];
+
+    const [xAxisLabelText, yAxisLabelText] = labels;
+    const graphInfo = {
+        range,
+        width,
+        height,
+    };
+    const [x1, y1] = vec.add(
+        pointToPixel(xAxisLabelLocation, graphInfo)[0],
+        [0, 0],
+    );
+    const [x2, y2] = vec.add(
+        pointToPixel(yAxisLabelLocation, graphInfo)[0],
+        [0, -24],
```

Comment:
It might be helpful to leave a comment here that explains what the -24 accomplishes.

Reply:
Thank you for pointing this out! Added a comment to explain. Originally this was a +1 on a coordinate point, so that might have been more clear. Please let me know if this explanation here is not clear though